graphql/server.go

```diff
@@ -220,15 +220,18 @@ func (c *conn) handleSubscribe(in *inEnvelope) error {
 			return output
 		})
 
-		output := runMiddlewares(middlewares, &ComputationInput{
-			Ctx:         ctx,
-			Id:          id,
-			ParsedQuery: query,
-			Previous:    previous,
-			Query:       subscribe.Query,
-			Variables:   subscribe.Variables,
-			Extensions:  in.Extensions,
-		})
+		computationInput := &ComputationInput{
+			Ctx:                  ctx,
+			Id:                   id,
+			ParsedQuery:          query,
+			Previous:             previous,
+			IsInitialComputation: previous == nil,
```

Comment:
can we change the mutation handling codepath as well? It should always return `true` I think.